Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM squidfunk/mkdocs-material
+RUN pip install mkdocs-material mkdocs-git-revision-date-localized-plugin mkdocs-redirects
```

Comment:
mkdocs-material and mkdocs-redirects are provided by squidfunk/mkdocs-material. So there is no need to install them.
`RUN pip install mkdocs-git-revision-date-localized-plugin`